caffe2/proto/caffe2.proto

```diff
@@ -46,20 +73,33 @@ message TensorProto {
   repeated double double_data = 9 [packed = true];
   // For int64
   repeated int64 int64_data = 10 [packed = true];
+  // For raw data
+  optional bytes raw_data = 15;
+  // External data by file name
```

Comment:
it's not only the file name, more like a record name

Reply:
Yes, will add more comments.